docs/source/concept_models/vivarium_census_synthdata/concept_model.rst

```diff
@@ -465,24 +484,41 @@ for changing an arc from a household in B to a new housing unit in C.
 
 I could imagine making these rates quite complex someday, to take into
 account the age, sex, race/ethnicity, household structure, and even
-past migration history.  But at this point, it is not clear how
-complex is necessary to have a successful synthetic data set for
-testing PRL algorithms, so we will keep it quite simple.
+past migration history.  At this point, it is clear that age is
+necessary to get the college dormatory migration right, so we might as
+well include sex and race/ethnicity stratification in the rates as
+well.
+
+A complex type of movement that we need to capture is moving into and
+out of Group Quarters; it is useful to think of six broad types of GQ
+for PRL purposes grouped into two categories: non-institutional
+(college, military, other non-institutional); and institutional
+(carceral, nursing homes, and other institutional).  College is likely
+to be the tough one in Census applications (Census will have SSN for
+most military and incarcerated, Medicare for most nursing home, but
+people living in dorms, especially who don't file their own tax
+returns might not be PIK-able.)
+
+To capture this, on the research side I will develop a migration rate
+file, with stratification columns for age group, sex, and
+race/ethnicity and data columns for the household move rate in moves
+per person year and individual move rate (also in moves per person
```

Comment:
I am assuming that these tables will be of the form:
- probability an individual moves given age/sex/other characteristics
- given that an individual is moving, probability they move into normal house, vs GQ1,...,6
- given that an individual is moving into housing type X, i will choose uniformly at random from all houses of type X. let me know if/when you want that probability updated/nuance added

Reply:
👍